<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoxTotalWid</class>
 <widget class="QWidget" name="BoxTotalWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>总电流</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>功率参数</string>
   </property>
  </widget>
  <widget class="QLabel" name="tApPowLab">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>50KW</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>有功功率</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>20</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>总电能</string>
   </property>
  </widget>
  <widget class="QLabel" name="tPowLab">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>48KVA:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>总电压</string>
   </property>
  </widget>
  <widget class="QLabel" name="tCurLab">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>100A</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="tEleLab">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>0.2Kwh</string>
   </property>
  </widget>
  <widget class="QLabel" name="tVolLab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>220V</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>54</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>视在功率:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
